src/libs/actions/Report.js

```diff
@@ -741,6 +750,26 @@ function subscribeToUserEvents() {
                 {error, pusherChannelName, eventName: Pusher.TYPE.REPORT_TOGGLE_PINNED},
             );
         });
+
+    // Live-update if a user has private domains listed as primary or secondary logins.
+    Pusher.subscribe(pusherChannelName, Pusher.TYPE.ACCOUNT_VALIDATED, (pushJSON) => {
+        Log.info(
+            `[Report] Handled ${Pusher.TYPE.ACCOUNT_VALIDATED} event sent by Pusher`,
+            false,
+            {isFromPublicDomain: pushJSON.isFromPublicDomain},
+        );
+        setIsFromPublicDomain(pushJSON.isFromPublicDomain);
+    }, false,
+    () => {
+        NetworkConnection.triggerReconnectionCallbacks('pusher re-subscribed to private user channel');
```

Comment:
Is there some reason we must call this here ?

Looks like we also do it here

https://github.com/Expensify/App/blob/57eed8bd0acdbc41359480fd31311c956d7bab07/src/libs/actions/Report.js#L744

and here

https://github.com/Expensify/App/blob/57eed8bd0acdbc41359480fd31311c956d7bab07/src/libs/actions/Report.js#L724

and here 

https://github.com/Expensify/App/blob/57eed8bd0acdbc41359480fd31311c956d7bab07/src/libs/actions/Report.js#L704

Reply:
Just a bad example copied over? We already call `triggerReconnectionCallbacks` when a user comes back online in `listenForReconnect`. All these calls seem unnecessary then. I can refactor that on v2 of this PR.

Reply:
Ah ok, maybe or we can create a follow up to look into a better way. Probably shouldn't need to block these changes. Just thought it was worth mentioning.